invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm_requests/menuTabInit.js

```diff
@@ -0,0 +1,3 @@
+import $ from "jquery";
+
+$('#request-community-submission-tab').tab();
```

Comment:
Make sure tab menus are accessible, see example: https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/index.js#L112

Reply:
the rule from the example you have linked is applied already, because it is connected with the class selector 
`$(.menu.tab)`
however, because there might be further changes, I will add this event to my element. 
In general I believe this part should be refactored, and placed in a common module, since it should be applied globally per theme